tests/repro_tests/averaging/translation/test_1dsfm_reproducibility.py

```diff
@@ -0,0 +1,77 @@
+"""Reproducibility tests for 1DSFM translation averaging.
+
+Authors: Ayush Baid
+"""
+import pickle
+import unittest
+from pathlib import Path
+from typing import List, Optional, Set, Tuple
+
+import gtsam
+from gtsam import BinaryMeasurementUnit3, BinaryMeasurementsUnit3, Point3, Pose3, Unit3
+
+import gtsfm.utils.geometry_comparisons as geometry_comparisons
+from gtsfm.averaging.translation.averaging_1dsfm import (
+    TranslationAveraging1DSFM,
+    HUBER_LOSS_K,
+    NOISE_MODEL_DIMENSION,
+    NOISE_MODEL_SIGMA,
+)
+from gtsfm.averaging.translation.translation_averaging_base import TranslationAveragingBase
+from tests.repro_tests.test_repro_base import ReproducibilityTestBase
+
+TEST_DATA_ROOT_PATH = Path(__file__).resolve().parent.parent.parent.parent / "data"
+RELATIVE_UNIT_TRANSLATIONS_PATH = (
+    TEST_DATA_ROOT_PATH / "reproducibility" / "inputs" / "relative_unit_translations_skydio32.pkl"
```

Comment:
Ayush, could you add a comment about whether the pkl files contain pickled GTSAM class instances, e.g. Rot3's, or are they numpy arrays?

Reply:
I'll add type hints where I load them.

Reply:
thanks, marking as resolved